tensorflow/compiler/tests/binary_ops_test.py

```diff
@@ -34,8 +34,7 @@
 from tensorflow.python.ops import math_ops
 from tensorflow.python.ops import nn_ops
 from tensorflow.python.platform import googletest
-from tensorflow.python.platform import test as test_lib
-
+from tensorflow.python.platform import test as test_lib  # pylint: disable=unused-import
```

Comment:
why not remove the `import` line?

Reply:
just out of concern it might cause a problem.  seems harmless to remove it, but not quite sure.